hpat/hiframes/hiframes_typed.py

```diff
@@ -851,7 +851,7 @@ def parse_impl(data):
 
     def _run_call_series(self, assign, lhs, rhs, series_var, func_name):
         # single arg functions
-        if func_name in ('sum', 'count', 'mean', 'var', 'min', 'max', 'prod'):
+        if func_name in ('sum', 'count', 'mean', 'min', 'max', 'prod'):
```

Comment:
I think this should brake parallelism. not sure.

Reply:
You are right, It breaks multiprocessing parallelism, but I still don't know how to save both approaches ("old" and "new") simultaneously. Please let me know if you have thoughts about it.

Reply:
where is no single and simple recipe

Reply:
As I can see there is no multiprocessing parallelism for the algorithm in HPAT, only multi-threading based on `numba.parfor`. Numba implemented the method for numpy also using `numba.parfor` and we invoke the method in our implementation. So we are able to drop "old style" in this case.

Reply:
May previous comment was wrong.